backend/routes/tweets.js

```diff
@@ -86,10 +86,7 @@ module.exports = function (DataHelpers) {
    *        description: Success
    *        content:
    *          application/json:
-   *            schema:
-   *              type: array
-   *              items:
-   *                $ref: '#/components/schemas/GetTweetResponseDto'
+   *                $ref: '#/components/schemas/TweetResponseDto'
```

Comment:
fix issues:
1) after "application/json:" the "schema:" is next child.
2) use whitespaces properly